### PR TITLE
Issue #2 fix: First time form validation failing fixed

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

File: editor/controllers/horizontal_controller.js [hunk 1]
```diff
@@ -98,7 +98,8 @@ angular.module('ftbApp', []).controller('ftbQuestionFormController', ['$scope',
   }
   $scope.formValidation = function() {
     $scope.submitted = true;
-    var formValid = $scope.ftbForm.$valid && /\[\[.*?\]\]/g.test($scope.ftbFormData.question.text);
+    var ftbFormValid = $scope.ftbFormData.question.text.length;
+    var formValid = (ftbFormValid > 0) && /\[\[.*?\]\]/g.test($scope.ftbFormData.question.text);
```
Comment (manojchandrashekar) [inline editor/controllers/horizontal_controller.js]: `ftbFormValid` has a numeric value? Why the comparison `> 0`?

Comment (JagadishPujari) [reply editor/controllers/horizontal_controller.js]: Yes numeric, refer above line 101. Checking question length.

Comment (manojchandrashekar) [reply editor/controllers/horizontal_controller.js]: :) Can we please rename the variable to `questionTextLength` or something?

**Comment by manojchandrashekar** [comment]
@vinukumar-vs FYA

**Pull request merged**